_assignments/mp1.md

```diff
@@ -302,22 +305,22 @@ Unity's [Scripting API Reference](https://docs.unity3d.com/ScriptReference/) is
 For the first script, we will detail the functions that we recommend,
 but for the others, we expect that you will refer to the API reference if needed.
 
-### Side Note: Input in VR
+#### Side Note: Input in VR
 
 Keyboards do not work well in VR, since the VR environment means that you cannot see the keyboard, and all of the keys largely feel the same.
 Controllers, with their contours and designated button shapes, are much easier to use blind.
 Refer to the following pages for more information, which you will need when you write the scripts below:
 
 - [Unity Input](https://docs.unity3d.com/Manual/ConventionalGameInput.html)
-- [Unity wiki page on Xbox controllers](http://wiki.unity3d.com/index.php?title=Xbox360Controller)
-- OVR Input: [general explanation](https://developer.oculus.com/documentation/unity/latest/concepts/unity-ovrinput),
+- [Unity Wiki page on Xbox controllers](http://wiki.unity3d.com/index.php?title=Xbox360Controller)

```

Comment:
Can we remove this since we're not using XBox controllers?

Reply:
Sure